lib/game.rb

```diff
@@ -104,11 +166,12 @@ def valid_user_coordinate?(coordinate)
   end
 
   def auto_generate_single_coordinate
-    possible = @user.board.cells.keys.shuffle
-    auto_cord = possible[0].to_s
-    possible.rotate!
-    auto_cord
+    next_hit = @user.board.cells.find_all do |cell|
```

Comment:
Thanks for putting in this guard!!! It is an awesome use of the enum find_all :)